### PR TITLE
Create Planetary Alignment

### PR DESCRIPTION
**Pull request by hickny**
All major planets will be visible on October 18, 2018 just after sunset from Sydney Australia.

### CODE REVIEW

**Comment by alex-w** [comment]
The last block you may change to `core.goHome();`.

The text is overlapping infostring and this is not very good.

**Comment by alex-w** [comment]
Would you describe by simple words what exactly you want to get by this script?

**Comment by gzotti** [comment]
This event is not a conjunction. It is a nice view, sure, but not a conjunction.
I also don't know why it must be the 18th, you see these objects +/-5days or so around that date.

File: scripts/Planetary Alignment [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+//
+//
+
+// The following two functions were copied from a script by Schdefon
+
+function blockText(textArray)
+{
+	var fontSize = 30;
+	var lineSep = fontSize * 1.5;
+	y = 250 + (lineSep*textArray.length);
```
Comment (hickny) [inline scripts/Planetary Alignment]: Change the 250 to 420 and the problem should be fixed.

**Comment by gzotti** [comment]
The final clearLabels(ids); must be removed to avoid a crash.

I recommend to place this script example to the website collection of script examples.

**Comment by hickny** [comment]
How do I place script to website collection?

**Pull request closed without merging**